.travis.yml

```diff
@@ -13,6 +12,7 @@ install:
   - pip install .[tests]
 script:
   - pytest --cov -rfsxEX
+  - pip install .[notebooks]
   - if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then ./scripts/test_notebooks.sh; fi
```

Comment:
If we drop 3.7 this needs to be updated to 3.7 as well (the reason for having `== 3.6` was to run the notebooks tests only once)